internal/engine/selectors/selectors_test.go

```diff
@@ -513,12 +729,45 @@ func TestSelectSelectorEntity(t *testing.T) {
 				},
 			},
 			selectorEntityBld: newTestPullRequestSelectorEntity(
+				newGithubProviderSelector(),
 				prWithProperties(map[string]any{
 					"github": map[string]any{"is_draft": true},
 				}),
 			),
 			selected: false,
 		},
+		{
+			name: "Use an artifact property that is defined and true result",
```

Comment:
these tests are not related to this PR, but the linter was yelling at me that the artifacts options are not used, so I added cases using them